src/allowlist/cloud-formation/allowlist.cfn.yaml

```diff
@@ -0,0 +1,361 @@
+  RestApi:
+    Properties:
+      EndpointConfiguration:
+        Types:
+        - 'EDGE'
+      Name: !Ref 'AWS::StackName'
+    Type: 'AWS::ApiGateway::RestApi'
+  # Deployment of Rest API gateway.
+  RestApiDeployment:
+    DependsOn:
+    - 'RestApiGet'
+    - 'RestApiPost'
+    # Use a nonce to force a new deployment when the API configuration changes.
+    Metadata:
+      Nonce: 5
+    Properties:
+      RestApiId: !Ref 'RestApi'
+      StageName: !Ref 'AWS::StackName'
+    Type: 'AWS::ApiGateway::Deployment'
+  # GET method for the REST API.
+  RestApiGet:
+    Properties:
+      ApiKeyRequired: false
+      AuthorizationType: 'NONE'
+      HttpMethod: 'GET'
+      Integration:
+        ConnectionId: !Ref 'VpcLink'
+        ConnectionType: 'VPC_LINK'
+        IntegrationHttpMethod: 'GET'
+        RequestParameters:
+          integration.request.path.requested_address:
+            'method.request.path.requested_address'
+        Type: 'HTTP_PROXY'
+        Uri: !Sub 'http://${Nlb.DNSName}/{requested_address}'
+      RequestParameters:
+        method.request.path.requested_address: true
+      ResourceId: !Ref 'RestApiRequestedAddress'
+      RestApiId: !Ref 'RestApi'
+    Type: 'AWS::ApiGateway::Method'
+  # POST method for the REST API.
+  RestApiPost:
+    Properties:
+      ApiKeyRequired: false
+      AuthorizationType: 'AWS_IAM'
+      HttpMethod: 'POST'
+      Integration:
+        ConnectionId: !Ref 'VpcLink'
+        ConnectionType: 'VPC_LINK'
+        IntegrationHttpMethod: 'POST'
+        RequestParameters:
+          integration.request.path.requested_address:
+            'method.request.path.requested_address'
+        Type: 'HTTP_PROXY'
+        Uri: !Sub 'http://${Nlb.DNSName}/{requested_address}'
+      RequestParameters:
+        method.request.path.requested_address: true
+      ResourceId: !Ref 'RestApiRequestedAddress'
+      RestApiId: !Ref 'RestApi'
+    Type: 'AWS::ApiGateway::Method'
+  # Path capture for the requested address in the REST API.
+  RestApiRequestedAddress:
+    Properties:
+      ParentId: !GetAtt 'RestApi.RootResourceId'
+      PathPart: '{requested_address}'
+      RestApiId: !Ref 'RestApi'
+    Type: 'AWS::ApiGateway::Resource'
+  # Route table for the VPC.
+  RouteTable:
+    Properties:
+      VpcId: !Ref 'VPC'
+    Type: 'AWS::EC2::RouteTable'
+  # A group containing all VPC subnets to be accessible for Redis.
+  SubnetGroup:
+    Properties:
+      SubnetGroupName: !Ref 'AWS::StackName'
+      SubnetIds:
+      - !Ref 'SubnetA'
+      - !Ref 'SubnetB'
+    Type: 'AWS::MemoryDB::SubnetGroup'
+  # Virtual private cloud for internal networking.
+  VPC:
+    Properties:
+      CidrBlock: !Ref 'VpcCidrBlock'
+      EnableDnsHostnames: true
+      EnableDnsSupport: true
+    Type: 'AWS::EC2::VPC'
+  # Connection to network load balancer.
+  VpcLink:
+    Properties:
+      Name: !Ref 'AWS::StackName'
+      TargetArns:
+      - !Ref 'Nlb'
+    Type: AWS::ApiGateway::VpcLink
```

Comment:
CI fails because of missing quotes here I think.

Reply:
Addressed in a106234